features/steps/data/__init__.py

```diff
@@ -10,10 +10,8 @@
                       "to install the latest version.")
 from . import util
 from .plugins import HtmlOutput, Markdown
-try:
-    import tidy
-except ImportError:
-    tidy = None
+
+tidy = None
```

Comment:
We will. I just wanted to get this up with a minimal incision so as to discuss with you.